kafka-connect-s3/src/test/java/io/confluent/connect/s3/DataWriterAvroTest.java

```diff
@@ -107,7 +106,7 @@ public void setUp() throws Exception {
     format = new AvroFormat(storage);
 
     s3.createBucket(S3_TEST_BUCKET_NAME);
-    assertTrue(s3.doesBucketExist(S3_TEST_BUCKET_NAME));
+    assertTrue(s3.doesBucketExistV2(S3_TEST_BUCKET_NAME));
```

Comment:
these changes (and similar ones within) will conflict with recent downstream changes. Can you please revert?